src/Uno.UI/LayoutProvider.Android.cs

```diff
@@ -53,8 +53,8 @@ internal void Start(View view)
 		{
 			if (view?.WindowToken != null)
 			{
-				_adjustNothingLayoutProvider.Start(view);
 				_adjustResizeLayoutProvider.Start(view);
+				_adjustNothingLayoutProvider.Start(view);				
```

Comment:
We should also call `Stop` when the view changes.

Reply:
Updated as we talked